flink-python/pyflink/table/tests/test_batch_table_api.py

```diff
@@ -0,0 +1,418 @@
+################################################################################
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+# #  distributed under the License is distributed on an "AS IS" BASIS,
+# #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# #  See the License for the specific language governing permissions and
+# # limitations under the License.
+################################################################################
+import os
+
+from pyflink.table.types import DataTypes
+from pyflink.table.window import Tumble, Slide, Session
+
+from pyflink.testing.test_case_utils import PyFlinkBatchTableTestCase
+
+
+class BatchTableTests(PyFlinkBatchTableTestCase):
```

Comment:
Good idea!

Reply:
Thanks for your reply @dianfu!
I think it again, you are right, add base test class which can run both Batch and Stream is pretty good, but from the points of CI time cost view,  for Python API wen only need check the `batchEvn` work well, and some special operators which only for batch need be check, other common operators test for stream is enough, because Python API only proxy to Java API, and Java API's test already cover the batch and stream. It's this make sense to you? @WeiZhong94 @dianfu

Reply:
That makes sense to me. I have reduced tests in the new commit.